File: app/src/main/java/com/doordash/binding/binder/MenuBinder.java
package com.doordash.binding.binder;

import com.doordash.binding.binder.common.ConditionalDataBinder;
import com.doordash.binding.viewmodel.DishModel;

public class MenuBinder extends ConditionalDataBinder<DishModel> {
    /**
     *
     * @param bindingVariable
     * @param layoutId
     */
    public MenuBinder(int bindingVariable, int layoutId){
        super(bindingVariable,layoutId);
    }

    @Override
    public boolean canHandle(DishModel model) {
        return true;
    }

}

File: app/src/main/java/com/doordash/binding/binder/MenuBinder.kt
package com.doordash.binding.binder

import com.doordash.bean.PopularItem
import com.doordash.binding.binder.common.ConditionalDataBinder

class MenuBinder(bindingVariable: Int, layoutId: Int) :
    ConditionalDataBinder<PopularItem>(bindingVariable, layoutId) {
    override fun canHandle(model: PopularItem): Boolean {
        return true
    }
}